frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/gitrepositoryselection/IGitRepositorySelectionComponent.java

```diff
@@ -0,0 +1,7 @@
+package com.virtuslab.gitmachete.frontend.ui.api.gitrepositoryselection;
+
+import javax.swing.JComponent;
+
+public interface IGitRepositorySelectionComponent extends IGitRepositorySelectionProvider {
```

Comment:
I hate to suggest that... but it's rather `IGitRepositorySelectionComponentProvider`

Reply:
![image](https://user-images.githubusercontent.com/3383210/85303177-70e95000-b4aa-11ea-921f-cccf57cf82a7.png)

Reply:
![image](https://user-images.githubusercontent.com/19799111/85313549-7fd6ff00-b4b8-11ea-9ed6-01a58a586bd8.png)

Reply:
![image](https://user-images.githubusercontent.com/19799111/85315290-10aeda00-b4bb-11ea-961f-46d935321da6.png)

Reply:
hmmm isn't it `IGitRepositorySelectionProviderComponentProvider` actually? :thinking:

Reply:
![fug](https://user-images.githubusercontent.com/3383210/85315739-e6a9e780-b4bb-11ea-8ebe-d5bcb6b34354.png)